ios/RCTWebRTC/WebRTCModule+RTCPeerConnection.m

```diff
@@ -309,6 +309,31 @@ @implementation WebRTCModule (RTCPeerConnection)
                 }];
 }
 
+RCT_EXPORT_METHOD(peerConnectionSendDTMF:(nonnull NSString *)tone duration:(NSTimeInterval)duration interToneGap:(NSTimeInterval)interToneGap objectID:(nonnull NSNumber *)objectID)
+{
+  RTCRtpSender* audioSender = nil ;
+  RTCPeerConnection *peerConnection = self.peerConnections[objectID];
+  if (!peerConnection) {
+    NSLog(@"peerConnectionSendDTMF() peerConnection is nil");
+    return;
+  }
+  for (RTCRtpSender *sender in peerConnection.senders){
+    if([[[sender track] kind] isEqualToString:@"audio"]) {
+       audioSender = sender;
+     } 
+  }
+  if(audioSender){
+    NSOperationQueue *queue = [[NSOperationQueue alloc] init];
+    [queue addOperationWithBlock:^{
+        BOOL istoneplayed = [audioSender.dtmfSender insertDtmf:tone duration:duration interToneGap:interToneGap];
```

Comment:
Is `dtmfSender` nillable?

Reply:
No, It should always exist in audioSender